gui/js/tray.window.js

```diff
@@ -155,7 +155,8 @@ module.exports = class TrayWM extends WindowManager {
       'auto-launcher': (event, enabled) => autoLaunch.setEnabled(enabled),
       'close-app': () => this.desktop.stopSync().then(() => this.app.quit()),
       'open-file': (event, path) => this.openPath(path),
-      'unlink-cozy': this.onUnlink
+      'unlink-cozy': this.onUnlink,
+      'manual-start-sync': () => this.desktop.synchronize('full')
```

Comment:
What @aenario suggests is to _stop_ any active synchronisation by using the same as 2 lines above before restarting a full sync:
https://github.com/cozy-labs/cozy-desktop/blob/edac2240bd920e9033c02fbc1a7be9fe1d125de2/gui/js/tray.window.js#L156